### PR TITLE
Sh dossier george bib

### PR DESCRIPTION
**Pull request by SusanneHummel**
I checked my changes again and did minor corrections. now it should be okay. 
I added the bibliography to the gadl and to colophon#1

### CODE REVIEW

File: 1001-2000/LIT1455GadlaG.xml [hunk 1]
```diff
@@ -51,52 +61,106 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
             <listBibl type="editions">
-               <bibl><ptr target="bm:Budge1930George"/></bibl>
+               <bibl>
+                  <ptr target="bm:Budge1930George"/>
+                  <citedRange unit="page">169–274</citedRange>
+               </bibl>
+               <bibl>
+                  <ptr target="bm:1967GadlaGiyorgis"/>
+                  <citedRange unit="page">21–93</citedRange>
+                  <citedRange unit="item">chapter 3–9</citedRange>
```
Comment (MassimoVilla) [inline 1001-2000/LIT1455GadlaG.xml:84]: chapters (also below)

File: 1001-2000/LIT1455GadlaG.xml [hunk 2]
```diff
@@ -51,52 +61,106 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                  <persName ref="PRS12752Mikael">ሚካኤል፡</persName>  ዘይንስ፡
-                  ወልደ፡ ወልዱ፡ ለ<persName ref="PRS12753Giyorgis"><roleName type="title">አባ፡</roleName>ጊዮርጊስ፡</persName> 
-                  ወልደ፡ አባ፡ <persName ref="PRS12754Mikale"><roleName type="title">አባ፡</roleName> ሚካኤል፡</persName> ጳጳስ፡ እንዘ፡ ይረድኦ፡ መንፈስ፡ ቅዱስ።</ab>
-               <note>Copied colophon of the translation, taken from <ref type="mss" corresp="BLorient712"></ref></note>
-               <div type="textpart" subtype="colophon" xml:id="coloph3"><ab> ወፍዳሜሁሰ፡ ኮነ፡፪ለወርኃ፡ ታስሪን፡ ቀደማይ፡ በሐሳቦሙ፡ ለዕብራውያን፡ ወበግዕዝ፡ አመ፡፱ዓውድ፡ ቀመር፡ ነ፩ጥንትዮን፡ ወበ፱ዓውደ፡ ዕለት። በ፱አበቅቴ፡ በካልአ፡ ዓመተ፡ መንግሥቱ፡ ለደዊት፡ ንጉሠ፡ አግዓዜ፡ ወልደ፡ ናዖድ፡ ንጉሠ፡ ድቱዓ፡ ሃይማኖት፡ በዕለተ፡ ሰኑይ፡ በ፲ሰዓት፡ ዘመዓልት፡ አመ፡ ከርመ፡ በምድረ፡ አዘዞ።
-               </ab>
-                   <note>The copied colophon (taken from <ref type="mss" corresp="BLorient713"></ref>) gives the precise date of translation, <date when="1510">2nd year of the reign of Dawit (that is, Lǝbna Dǝngǝl), who began his reign in August 1508</date> </note>
+            <div type="textpart" subtype="incipit">
+               <ab> ገድል፡ ወስምዕ፡ ዘቅዱስ፡ ወብፁዕ፡ ሰማዕት፡ ዐቢይ፡ ማር፡ ጊዮርጊስ፡ etc. ዘፈጸመ፡ ገድሎ፡ አመ፡ ፪ወ፫፡ ለወርኃ፡
```
Comment (MassimoVilla) [inline 1001-2000/LIT1455GadlaG.xml:118]: I think "etc." should be encoded differently. In the guidelines (https://betamasaheft.eu/Guidelines/?q=erased&id=text-encoding) the use of `<gap reason="ellipsis"/>` is recommended for portions of text omitted from the edition by the editor, for whatever reason (brevity, context, language, etc.)

File: 1001-2000/LIT1455GadlaG.xml [hunk 3]
```diff
@@ -51,52 +61,106 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                  መርሙደ፡ ዘውእቱ፡ ሚያዝያ።</ab>
+            </div>
+            <div type="textpart" subtype="colophon" xml:id="coloph1">
+               <ab>ወሶበ፡ ፈጸመ፡ ቅዱስ፡ ጊዮርጊስ፡ ገድሎ፡ አነ፡ ሰቅራጥስ፡ ገብሩ፡ ለጊዮርጊስ፡ ኮንኩ፡ እትልዎ፡ ለእግዚእየ፡ ጊዮርጊስ፡ በኵሉ፡
+                  {ገድሉ}፡ ወፃማሁ፡ ወለካልሁ፡ ዘንተ፡ ጸሐፍኩ፡ … ተፈጸመ፡ እንዘ፡ ንነግድ፡ ብሔረ፡ ቈስጠንጢኖስ፡ መካን፡ ዘስሙ፡ ቍስቋስ፡
```
Comment (MassimoVilla) [inline 1001-2000/LIT1455GadlaG.xml:123]: What is the meaning of these brackets { } ? I think they might be replaced with a suitable tag. Give a look at the guidelines: https://betamasaheft.eu/Guidelines/?q=erased&id=text-encoding.

File: 1001-2000/LIT1455GadlaG.xml [hunk 4]
```diff
@@ -51,52 +61,106 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+            <div type="textpart" subtype="colophon" xml:id="coloph2">
+               <ab>ሃሌ፡ ሉያ፡ ተተርጐመት፡ ዛቲ፡ መጽሐፍ፡ እምልሳነ፡ ዐረቢ፡ ለግዕዝ፡ በአፈ፡ <persName ref="PRS12752Mikael"
+                     >ሚካኤል፡</persName> ዘይንስ፡ ወልደ፡ ወልዱ፡ ለ<persName ref="PRS12753Giyorgis"><roleName
+                        type="title">አባ፡</roleName>ጊዮርጊስ፡</persName> ወልደ፡ አባ፡ <persName
+                     ref="PRS12754Mikale"><roleName type="title">አባ፡</roleName> ሚካኤል፡</persName>
+                  ጳጳስ፡ እንዘ፡ ይረድኦ፡ መንፈስ፡ ቅዱስ።</ab>
+               <note>Copied colophon of the translation, taken from <ref type="mss"
+                     corresp="BLorient712"/></note>
+               <div type="textpart" subtype="colophon" xml:id="coloph3">
+                  <ab> ወፍዳሜሁሰ፡ ኮነ፡ ፪ለወርኃ፡ ታስሪን፡ ቀደማይ፡ በሐሳቦሙ፡ ለዕብራውያን፡ ወበግዕዝ፡ አመ፡ ፱ዓውድ፡ ቀመር፡ ነ፩ጥንትዮን፡
```
Comment (MassimoVilla) [inline 1001-2000/LIT1455GadlaG.xml:154]: `<ab>ወፍጻሜሁሰ፡`

File: 1001-2000/LIT1455GadlaG.xml [hunk 5]
```diff
@@ -51,52 +61,106 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                     ወበ፱ዓውደ፡ ዕለት። በ፱አበቅቴ፡ በካልአ፡ ዓመተ፡ መንግሥቱ፡ ለደዊት፡ ንጉሠ፡ አግዓዜ፡ ወልደ፡ ናዖድ፡ ንጉሠ፡ ድቱዓ፡
+                     ሃይማኖት፡ በዕለተ፡ ሰኑይ፡ በ፲ሰዓት፡ ዘመዓልት፡ አመ፡ ከርመ፡ በምድረ፡ አዘዞ። </ab>
+                  <note>The copied colophon (taken from <ref type="mss" corresp="BLorient713"/>)
+                     gives the precise date of translation, <date when="1510">2nd year of the reign
+                        of Dawit (that is, Lǝbna Dǝngǝl), who began his reign in August 1508</date>
```
Comment (MassimoVilla) [inline 1001-2000/LIT1455GadlaG.xml:159]: Dāwit

File: 5001-6000/LIT5464DiscourseofTheodotus.xml [hunk 6]
```diff
@@ -25,33 +27,64 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
         <body>
             <div type="bibliography">
-              </div>
-         <!---->
+                
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:1967GadlaGiyorgis"/>
+                        <citedRange unit="page">11–12</citedRange>
+                        <citedRange unit="item">chapter 1–2</citedRange>
```
Comment (MassimoVilla) [inline 5001-6000/LIT5464DiscourseofTheodotus.xml:66]: I would say "chapters"

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you @SusanneHummel !!

File: 1001-2000/LIT1455GadlaG.xml (hunk 2 above)
Comment (SusanneHummel) [reply 1001-2000/LIT1455GadlaG.xml:118]: This was already in the entry. I simply added the bibliographic references. I decided to restrict myself to what we need in our catalogue, this are just the refernces. Therefore, I left the encoding as it was.

File: 1001-2000/LIT1455GadlaG.xml (hunk 3 above)
Comment (SusanneHummel) [reply 1001-2000/LIT1455GadlaG.xml:123]: Same as before. The etxt was typed in and encoded by Zhenia.

File: 1001-2000/LIT1455GadlaG.xml (hunk 4 above)
Comment (SusanneHummel) [reply 1001-2000/LIT1455GadlaG.xml:154]: I will correct it (together with other typos).

**Comment by SusanneHummel** [comment]
Thank you Massimo!

File: 1001-2000/LIT1455GadlaG.xml (hunk 2 above)
Comment (MassimoVilla) [reply 1001-2000/LIT1455GadlaG.xml:118]: Sorry, I thought the entire encoding was yours.

**Review by thea-m — APPROVED** [review]

**Comment by SusanneHummel** [comment]
Thank you Dorothea! I will do the changes next week. This week I work from home without functional Github on my laptop.

**Comment by SusanneHummel** [comment]
Massimo, could you please merge it (if you agree) ? Since you requested changes, I cannot do it. Thank you!

**Review by MassimoVilla — APPROVED** [review]

**Pull request merged**